pkg/operator/README.md

```diff
@@ -65,6 +65,13 @@ Make sure KUBECONFIG is set:
 ```sh
 make admin.kubeconfig
 export KUBECONFIG=$(pwd)/admin.kubeconfig
+```
+If you are using a private cluster, you need to connect to the respective VPN of your region. For example for eastus:
+```sh
+openvpn --config secrets/vpn-eastus.ovpn
```

Comment:
Should we be putting a `sudo` in front of this command as typically you need sudo to run `openvpn`?